src/flightlog_fields_presenter.js

```
@@ -2239,6 +2283,39 @@ FlightLogFieldPresenter.decodeDebugFieldToFriendly = function (
+      case "FEEDFORWARD_LIMIT":
+        switch (fieldName) {
+          case "debug[0]": // jitter attenuator percent
+            return `${(value / 100).toFixed(2)} %`;
+          // case "debug[1]": max setpoint rate for the axis
+          // case "debug[2]": setpoint
+          // case "debug[3]": feedforward * 0.01
+          // case "debug[4]": setpoint speed un-smoothed * 0.01f
+          // case "debug[5]": setpoint speed smoothed * 0.01f
+          // case "debug[6]": feedforward smoothing PT1K * 1000
+            return `${(value / 10).toFixed(1)}`;
+          case "debug[7]":   // smoothed RxRateHz
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)